pyproject.toml

```diff
@@ -1,5 +1,52 @@
 [build-system]
-requires = ["setuptools>=56", "setuptools_scm[toml]>=3.4.1"]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.1"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
```

Comment:
I thought we settled on leaving this at the bottom, no?

Reply:
Oh.

> though it will make the transition more difficult

I must have misunderstood. Reverting 792b3a2 then.